packages/pluggableWidgets/datagrid-web/src/components/__tests__/__snapshots__/Table.spec.tsx.snap

```diff
@@ -28,7 +28,7 @@ exports[`Table renders the structure correctly 1`] = `
           Object {
             "Cell": [Function],
             "Footer": [Function],
-            "Header": "Test",
+            "Header": "[Empty caption]",
```

Comment:
1. Based on skimming this relatively quickly, I only see changes for `[Empty caption]` but not for the other changes.
2. It changes from `Test` to `[Empty caption]`, which I feel like shouldn't happen? "Test" isn't an empty caption? 😄

Reply:
Wrong logic indeed, just switched.